vassal-app/src/main/java/VASSAL/tools/IconButton.java

```diff
@@ -33,6 +34,8 @@ public static Color getDefaultColor(int type) {
       return Color.green;
     case CROSS_ICON:
       return Color.red;
+    case GREY_MINUS_ICON:
+      return Color.lightGray;
```

Comment:
What's the reason for making a new icon? I thought you were trying to disable the back button---which I don't see happening anywhere.

Reply:
I've "greyed-out" and disabled the back button so as not to ugly-up the UI. I started by just removing the button altogether, but thought it doesn't look right and re-worked it as a greyed-out control, which, to my eyes, looks better. The disabling code is definitely in there. I've tested it. You will see conditional execution wrapping the button creation process and the "doPrev" method.

Reply:
If you run the code, you will see the final setting in the editor is now a boolean check box that sets it to go forward only, disabling the backward functionality at the designer's discretion.

Reply:
I've also put in comments to mark my code...search for // FWD_ONLY

Reply:
I don't find a call to `setEnabled(false)` anywhere in the diff. That's what would be needed to disable the back button.

Reply:
That's probably what someone who knows Java Swing would do. The way I did it was on line 1025 of  TurnTracker.java.

Reply:
Ok, so the canonical way to do this would be (1) to leave the action triggered by the button (`doPrev()`) unaltered, and (2) call `setEnabled(!fwdOnly)` on the button.

Reply:
Roger that. I will refactor. Please feel free to add anything else to the list. I'll probably push the new commit tonight.